src/app/components/simulation/animation-view.scss

```diff
@@ -1,4 +1,11 @@
-@import "./sprite.scss";
+@import "../vars.scss";
+
+@function cmToPx($cm) {
+  // the 0 cm line is 327px from the top of the container.
+  $cm0: 327px;
+  // there are 12.2px between each cm line.
+  @return calc($cm0 - calc(12.2px * $cm));
```

Comment:
No to need change this, but I *think* that since this is an .scss file you can skip the runtime `calc()` call and just do this:

`@return $cm0 - (12.2px * $cm)`